packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -224,6 +233,15 @@ cp bin/* %{buildroot}%{_sbindir}/
 cp extra/katello-tracer-upload-dnf %{buildroot}%{_sbindir}/katello-tracer-upload
 %endif
 
+%if 0%{?suse_version} >= 1500
+sed -i 's/bin\/python$/bin\/python3/' %{buildroot}%{_sbindir}/katello-enabled-repos-upload
+sed -i 's/bin\/python$/bin\/python3/' %{buildroot}%{_sbindir}/katello-package-upload
+sed -i 's/bin\/python$/bin\/python3/' %{buildroot}%{_sbindir}/katello-tracer-upload
+sed -i 's/bin\/python$/bin\/python3/' %{buildroot}%{plugins_dir}/enabled_repos_upload.py
```

Comment:
Do the zypper plugins really need to specify the interpreter to work?

Reply:
They can be written with different languages (bash, python). 
Without specification in first line:

2022-01-17 16:00:09 <5> sles12sp5(2284) [zypp] Exception.cc(log):166  - [11-Die Ressource ist zur Zeit nicht verfügbar]
2022-01-17 16:00:09 <5> sles12sp5(2284) [zypp] Exception.cc(log):166 
2022-01-17 16:00:09 <2> sles12sp5(2284) [PLUGIN] PluginScript.cc(~PluginDumpStderr):75 ! /usr/lib/zypp/plugins/commit/package_upload.py: line 13: from: command not found
2022-01-17 16:00:09 <2> sles12sp5(2284) [PLUGIN] PluginScript.cc(~PluginDumpStderr):75 ! /usr/lib/zypp/plugins/commit/package_upload.py: line 14: from: command not found
2022-01-17 16:00:09 <2> sles12sp5(2284) [PLUGIN] PluginScript.cc(~PluginDumpStderr):75 ! /usr/lib/zypp/plugins/commit/package_upload.py: line 15: import: command not found
2022-01-17 16:00:09 <2> sles12sp5(2284) [PLUGIN] PluginScript.cc(~PluginDumpStderr):75 ! /usr/lib/zypp/plugins/commit/package_upload.py: line 16: import: command not found
2022-01-17 16:00:09 <2> sles12sp5(2284) [PLUGIN] PluginScript.cc(~PluginDumpStderr):75 ! /usr/lib/zypp/plugins/commit/package_upload.py: line 18: from: command not found
2022-01-17 16:00:09 <2> sles12sp5(2284) [PLUGIN] PluginScript.cc(~PluginDumpStderr):75 ! /usr/lib/zypp/plugins/commit/package_upload.py: line 20: from: command not found